make/modules/jdk.jdwp.agent/Lib.gmk

```diff
@@ -47,6 +47,27 @@ TARGETS += $(BUILD_LIBDT_SOCKET)
 
 ################################################################################
 
+# SapMachine 2024-02-16: Add dt_filesocket implementation
+$(eval $(call SetupJdkLibrary, BUILD_LIBDT_FILESOCKET, \
+    NAME := dt_filesocket, \
+    OPTIMIZATION := LOW, \
```

Comment:
LOW is not really low (-O2 on gcc). It's the default for all created libs. Since the performance is dominated by the systemcalls anyways, going higher makes no sense.